options/options.go

```diff
@@ -363,6 +366,12 @@ func (o *Options) CLI() serpent.OptionSet {
 			Value:       serpent.StringOf(&o.GitSSHPrivateKeyPath),
 			Description: "Path to an SSH private key to be used for Git authentication.",
 		},
+		{
+			Flag:        "git-ssh-private-key-base64",
+			Env:         WithEnvPrefix("GIT_SSH_PRIVATE_KEY_BASE64"),
+			Value:       serpent.StringOf(&o.GitSSHPrivateKeyBase64),
+			Description: "SSH private key to be used for Git authentication.",
```

Comment:
```suggestion
			Description: "Base64 encoded SSH private key to be used for Git authentication. This option takes precedence over the private key path option.",
```
The "This option takes ..." is just there for completeness, may be rewritten depending on how the final implementation turns out.

Reply:
Yep, we should be explicit that both options are mutually exclusive.

Reply:
Have documented this now 👍